src/client.ts

```diff
@@ -774,9 +736,9 @@ export class SpvWalletClient {
    * const tx = await spvWalletClient.RecordTransaction(finalized, draft.id, metadata)
    */
   async SendToRecipients(recipients: Recipients, metadata: Metadata): Promise<Transaction> {
-    const draft = await this.DraftToRecipients(recipients, metadata)
-    const finalized = this.FinalizeTransaction(draft)
-    return this.RecordTransaction(finalized, draft.id, metadata)
+    const draft = await this.DraftToRecipients(recipients, metadata);
+    const finalized = this.FinalizeTransaction(draft);
```

Comment:
IMO name `FinalizeTransaction` is misleading, maybe we could change it to `SignTransaction`? WDY?

Reply:
It's not my methods 🙈 
However I agree with you, so `SignTransaction` let it be 😄

Reply:
We should remember about this change and apply it later on go-client